$NetBSD: patch-configure,v 1.4 2018/05/10 07:12:53 adam Exp $

Remove code that insists wx-config is in a directory named bin

--- configure.orig	2014-12-15 15:10:17.000000000 +0000
+++ configure
@@ -6467,15 +6467,6 @@ if test "${with_wx+set}" = set; then :
 
 else
 
-		WX_HOME=/usr/local/wx2
-		if test ! -f "${WX_HOME}/bin/wx-config"
-		then
-			WX_HOME=/usr/local
-			if test ! -f "${WX_HOME}/bin/wx-config"
-			then
-				WX_HOME=/usr
-				if test ! -f "${WX_HOME}/bin/wx-config"
-				then
 		    # Search the path
 				    for ac_prog in wx-config
 do
@@ -6528,10 +6519,6 @@ done
 		    else
 					   WX_HOME=`${WX_CONFIG} --prefix`
 		    fi
-				fi
-			fi
-		fi
-		WX_CONFIG=${WX_HOME}/bin/wx-config
 
 fi
 
@@ -6558,9 +6545,8 @@ $as_echo "failed" >&6; }
 $as_echo "ok" >&6; }
 
 
-	if test -n "${WX_HOME}"
+	if test -n "${WX_CONFIG}"
 	then
-		LDFLAGS="$LDFLAGS -L${WX_HOME}/lib"
 		WX_OLD_LDFLAGS="$LDFLAGS"
 		WX_OLD_CPPFLAGS="$CPPFLAGS"
 
